examples/step-12/doc/builds-on

```diff
@@ -1 +1 @@
-step-7 step-39
+step-39
```

Comment:
I am not sure about this. I would probably remove 39 instead as the prerequisite for 12 instead. I have plans to introduce a new step in between anyways. Can you remove this for now?

Reply:
ok!